### ISSUE TITLE
standalone semantics including ffi

### ISSUE THREAD

**Issue by xrchz** [CakeML/cakeml#56]
add whatever is required to semantics/standaloneTheory to make a complete top-level semantics for whole CakeML programs (including FFI)

**Comment by xrchz** [comment]
probably depends on #60

**Comment by xrchz** [comment]
@SOwens shall we consider this done for now, with the definition of `sem` in `standaloneTheory`? (The composed system stuff can be deferred for a while.)
